docs/userdoc.qmd

```diff
@@ -1,14 +1,21 @@
 # iBridges User Documentation
 ## Synopsis 
-iBridges is a GUI to an Data management platform based on iRODS. It exposes iRODS functionality through a GUI to the users. We divided the different function into groups where each function group is represented in an own tab in the GUI. The tabs are configured for the user depending on the user's rights and role in the Data management platform.
+iBridges is a GUI to a Data management platform based on iRODS. It exposes iRODS functionality through the graphical interface to the users. We divided the different functions into groups where each function group is represented in an own tab in the GUI. The tabs are configured for the user depending on the user's rights and role in the Data management platform.
+
+The tabs which currently are in use:
+
 - The **Browser** is a tool to browse through iRODS collections and gather information on iRODS data objects
 - The **Up and Download** tab gives tools to up and download large data safely.
 - The **ELN Data upload** tab is made for specific data uploads that belong to an Experiment described in an ElabJournal instance. 
-- In the **Compress/bundle data** tab the user is given tools to bundle (tar) or compress (zip) iRODS data collections and to extract them again.
```

Comment:
Why remove _bundle_?  _Compress_ functionality is gone, but _bundle_ functionality remains.